packages/base/src/Base.ts

```diff
@@ -163,3 +163,49 @@ export function SpectrumMixin<T extends Constructor<ReactiveElement>>(
 }
 
 export class SpectrumElement extends SpectrumMixin(LitElement) {}
+
+if (window.__swc.DEBUG) {
+    window.__swc = {
+        ...window.__swc,
+        issuedWarnings: new Set(),
```

Comment:
If the goal is to shorten, then consider just `warnings`, since `emitted` is so general as to not be self-descriptive.

Reply:
`issued*` or some other past tense verb is I think it important because this is where we track what _has_ happened. I also like the parallel to `console.*` in the management of the API names here. I could see it making sense to expanding into those other types as we find good use cases for them.